src/maliput_malidrive/builder/road_geometry_builder.cc

```diff
@@ -283,9 +283,13 @@ RoadGeometryBuilder::LaneConstructionResult RoadGeometryBuilder::BuildLane(
                              allow_semantic_errors /* allow_negative_width */);
   // When semantic errors aren't allowed G1 contiguity must be enforced for all lanes.
   // Otherwise, only drivable lanes are enforced.
-  const bool enforce_contiguity = !allow_semantic_errors || is_driveable_lane(*lane);
+  const bool enforce_contiguity_non_driveable_lanes = !allow_semantic_errors || is_driveable_lane(*lane);
+  // An exception for the case above, where lane widths' C1 discontinuities are tolerated and then fixed by adding an
```

Comment:
Avoid saying that C1 discontinuities are tolerated because we do not actually. Mention that we adapt if discontinuities are found.

Reply:
Done.